docs/photoniq/vwrs/get-started-vwrs/index.md

```diff
@@ -0,0 +1,22 @@
+---
+sidebar_position: 10
+title: Get Started with VWRs
+---
+
+Macrometa PhotonIQ Virtual Waiting Rooms (VWRs) are essential tools for managing high traffic volumes on websites and applications, ensuring server stability and enhancing user experiences during peak times. By implementing VWRs, you can prevent server overload and potential downtime, which is crucial for maintaining service availability and user satisfaction.
+
+## App Integration Types
+
+Different types of applications require distinct approaches to integrate with Virtual Waiting Rooms (VWRs). Depending on the application's platform and user interaction model, the integration process can vary significantly. The paragraphs below explain how both non-native and native applications can effectively use VWRs to manage traffic and enhance user experiences.
```

Comment:
We are leaning heavily into the term "non-native".  While technically correct, it is not a common term that people might recognize.  Maybe go with "Web app".  

"... how both web apps and native applications can effectively..."